### PR TITLE
hidden of buttons when URL doesnt match echo360

### PR DESCRIPTION
**Pull request by ericjiang97**

### CODE REVIEW

File: app/src/popup.html [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <!doctype html>
 <html>
   <head>
-    <title>Getting Started Extension's Popup</title>
-	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <title>eEcho360 Download Manager</title>
```
Comment (darvid7) [inline app/src/popup.html:4]: typo?

File: app/src/popup.js [hunk 2]
```diff
@@ -89,6 +89,19 @@ document.addEventListener('DOMContentLoaded', function() {
     });
   }, false);
 
+  /*
```
Comment (darvid7) [inline app/src/popup.js:92]: why was this merged?

**Comment by ericjiang97** [comment]
Lol.. That was merged in 2 years ago

File: app/src/popup.html (hunk 1 above)
Comment (ericjiang97) [reply app/src/popup.html:4]: Technically the user can't see this

File: app/src/popup.js (hunk 2 above)
Comment (ericjiang97) [reply app/src/popup.js:92]: Idk... It was 2 years ago

**Pull request merged**